src/services/riotApiClient.js

```diff
@@ -75,6 +75,15 @@ class RiotApiClient {
       throw err
     }
   }
+  async fetchTFTMatchHistory(puuid, count){
+    
+    const matchIds = await this.getRecentMatches(puuid, count)
+    const matchListInfo = []
+        let i;
+        for(i = 0; i < matchIds.length; i++){
+             matchListInfo.push( await this.getMatchData(matchIds[i]))
+        } return matchListInfo
```

Comment:
Move the return to its own line:
```suggestion
       for(i = 0; i < matchIds.length; i++){
             matchListInfo.push( await this.getMatchData(matchIds[i]))
        } 
        
        return matchListInfo
```

Once you get this working, we will rework this part of the code to fetch all matches at once using Promise.all